src/utils.js

```diff
@@ -213,6 +216,14 @@ export function isMultiSelect(schema) {
   return Array.isArray(schema.items.enum) && schema.uniqueItems;
 }
 
+export function isFixedItems(schema) {
+  return Array.isArray(schema.items) && schema.items.length > 0 && schema.items.every(item => isObject(item));
+}
+
+export function allowAdditionalItems(schema) {
+  return isObject(schema.additionalItems);
+}
```

Comment:
> For the detail should we display a message in the error list, or just showing a warning in the console?

A warning in the console sounds good enough to me. I'd like to keep error reporting for strict jsonschema validation stuff, not lib limitations.